local_sales/app/controllers/companies_controller.rb

```diff
@@ -0,0 +1,60 @@
+class CompaniesController < ApplicationController
+  before_action :set_company, only: %i[show edit update destroy]
+
+  def index
+    @companies = Company.all
+  end
+
+  def show
+  end
+
+  def new
+    @company = Company.new
+  end
+
+  def edit
+  end
+
+  def create
+    @company = Company.new(company_params)
+
+    respond_to do |format|
+      if @company.save
+        format.html { redirect_to @company, notice: "Company was successfully created." }
+        format.json { render :show, status: :created, location: @company }
+      else
+        format.html { render :new, status: :unprocessable_entity }
+        format.json { render json: @company.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  def update
+    respond_to do |format|
+      if @company.update(company_params)
+        format.html { redirect_to @company, notice: "Company was successfully updated." }
+        format.json { render :show, status: :ok, location: @company }
```

Comment:
Same here. Shouldn't be `company`? I mean, RESTful APIs usually respond with the object after creating or updating it, not a location.